setup.py

```diff
@@ -78,6 +95,22 @@
             'alldeps': (
```

Comment:
I am wondering if we can have several key in the `dict`. We did that in `imbalanced-learn` and I find quite readable.

```python
extra_require={
    "tests": (
        f"pytest>={PYTEST_MIN_VERSION}",
        ...
    ),
    "docs": (
        f"matplotlib>={MATPLOTLIB_MIN_VERSION}",
        ...
    ),
}
```

Reply:
I got them changed, let me know what you think of it.